website/docs/docs/build/snapshots.md

```diff
@@ -295,6 +444,27 @@ Snapshots can be configured from both your `dbt_project.yml` file and a `config`
 
 Note: BigQuery users can use `target_project` and `target_dataset` as aliases for `target_database` and `target_schema`, respectively.
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+| Config | Description | Required? | Example |
+| ------ | ----------- | --------- | ------- |
+| Config | Description | Required? | Example |
+| ------ | ----------- | --------- | ------- |
+| [database] | The database that dbt should render the snapshot table into | No | analytics |
```

Comment:
Not quite right - see https://docs.getdbt.com/reference/resource-configs/database